lib/Elastica/Bulk/Action/IndexDocument.php

```diff
@@ -39,5 +41,17 @@ protected function _getMetadata(AbstractUpdateAction $action): array
             'parent',
             'retry_on_conflict',
         ], true);
+
+        $newOptions = \array_intersect_key($options, [
+            '_index' => 1,
+            '_type' => 1,
+            '_id' => 1,
+        ]);
+
+        foreach (\array_diff_key($options, $newOptions) as $k => $v) {
```

Comment:
Couldn't we just rename the options as they should be in 7.0?

Reply:
ok, force pushed a second version of the patch. I had to make breaking changes.